README.md

```diff
@@ -1,68 +1,51 @@
-# EVE
 
-This project aims to simulate basic lifeforms evolving and getting smarter, in a simplified manner.
+# EVE
 
-## Development
+[![experimental](http://badges.github.io/stability-badges/dist/experimental.svg)](http://github.com/badges/stability-badges)[![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-### Setup
+##### Simulate simple, artificial lifeforms as they learn and evolve
 
-1. Install cmake & [Catch2](https://github.com/catchorg/Catch2) on your OS
-2. Clone this repository:  
-   HTTPS: `git clone https://github.com/Xerus2000/eve`  
-   SSH: `git clone git@github.com:Xerus2000/eve.git`
-3. Build 'eve' via CLion so that all tasks become available for running
-4. In CLion go to `Settings > Editor > File and Code Templates` and remove the "Created by" header in c/cpp files
```

Comment:
missing as well

Reply:
I do not want to enforce using a specific IDE, nor give any IDE-specific information.
This file is supposed to contain important information you should find on short notice, not full instruction about how to do what.
For more detailed description, I'd rather link them to notion or something; the README might get unreadable pretty quick

Reply:
I disagree. Notion shouldn't be involved here at all. We have established pretty clearly that CLion is required. Sometimes you have to make assumptions. Who would be that mysterious outsider anyways? Any personas?
You could at least put it under Collaboration.

Reply:
I strongly dislike the idea of enforcing something on people, like a specific IDE for contributors.
Even though it might be annoying if people use different IDEs and have everything set up differently, the code they write is still the same. After all, we *want* people to collaborate, not exclude them base on their IDE

Reply:
I mean
only half of our active developers are using CLion, anyways